stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss

```diff
@@ -3,167 +3,153 @@
       @include devicebreak(mediumonlytab) {
-        display: none;
+        display: flex;
+
+        .dropdown-language {
+          display: none;
+        }
       }
-      li{
-        margin: 0 2.1875rem;
+
+      .dropdown-language {
+        margin-left: $spacing-12;
       }
     }
 
-    .logo {
+    .mid-nav {
+      display: flex;
+      justify-content: center;
       position: relative;
-      z-index: 1;
-      @extend %img_none;
-
       @include devicebreak(xlarge) {
-        @include transform(translateX(-50%));
-
+        height: 7.1875rem;
+      }
+      nav{
+        @include devicebreak(mediumonlytab) {
+          height: 4.6875rem;
+        }
+      }
+      ul {
+        position: absolute;
         left: 50%;
-        position: fixed;
+        width: 50%;    
+        top: 50%;
+        transform: translateY(-50%);
+        padding-left: 4.375rem;
+        display: flex;
+        height: 100%;
+        align-items: center;
         z-index: 9;
-      }
 
-      @include devicebreak(mediumonlytab) {
-        height: 4.6875rem;
-        margin: 0;
-      }
+        &:first-child{
+          right: 50%;
+          left: inherit;
+          justify-content: right;
+          padding-right: 4.375rem;
+          padding-left: 0;
+        }
 
-      img {
-        @extend %logo;
+        @include devicebreak(mediumonlytab) {
+          display: none;
+        }
+        li{
+          margin: 0 2.1875rem;
+        }
       }
-    }
-  }
 
-  .align-content-middle {
-    align-items: center;
-    position: relative;
-  }
+      .logo {
+        position: relative;
+        z-index: 1;
+        @extend %img_none;
 
-  .top-right {
-    display: flex;
-    justify-content: flex-end;
+        @include devicebreak(xlarge) {
+          @include transform(translateX(-50%));
 
-    @include devicebreak(mediumonlytab) {
-      .icon-globe {
-        display: none;
+          left: 50%;
+          position: fixed;
+          z-index: 9;
+        }
+
+        @include devicebreak(mediumonlytab) {
+          height: 4.6875rem;
+          margin: 0;
+        }
+
+        img {
+          @extend %logo;
+        }
       }
     }
 
-    .icon-globe {
-      background: none;
-      border: 0;
-      display: inline-flex;
-      padding: 0;
+    .align-content-middle {
+      align-items: center;
+      position: relative;
+    }
+
+    .top-right {
+      display: flex;
+      justify-content: flex-end;
+
       @include devicebreak(mediumonlytab) {
-        display: none;
+        .icon-globe {
+          display: none;
+        }
       }
-    }
 
-    .icon-search {
-      background: none;
-      border: 0;
-      display: inline-flex;
-      margin-left: $spacing-16;
-      margin-right: $spacing-12;
-      padding: 0;
+      .icon-globe {
+        background: none;
+        border: 0;
+        display: inline-flex;
+        padding: 0;
+        @include devicebreak(mediumonlytab) {
+          display: none;
+        }
+      }
```

Comment:
@Ashish-srij These are interactive buttons and need to have a cursor pointer. Right now the cursor is default. Please change.

Reply:
@shishirphadke Done.